ballot/models.py

```diff
@@ -352,6 +352,121 @@ def update_or_create_ballot_item_for_polling_location(
+                                     google_civic_election_id, defaults):
+        """
+        Create BallotItem table entry with BallotItem details
+        :param ballot_item_display_name:
+        :param local_ballot_order:
+        :param state_code:
+        :param google_civic_election_id:
+        :param defaults:
+        :return:
+        """
+
+        new_ballot_item_created = False
+        new_ballot_item = ''
+
+        try:
+            new_ballot_item = BallotItem.objects.create(
+                ballot_item_display_name=ballot_item_display_name,
+                local_ballot_order=local_ballot_order,
+                state_code=state_code,
+                google_civic_election_id=google_civic_election_id)
+            if new_ballot_item:
+                success = True
+                status = "CONTEST_OFFICE_CREATED"
+                new_ballot_item_created = True
+                new_ballot_item.contest_office_id = defaults['contest_office_id']
+                new_ballot_item.contest_office_we_vote_id = defaults['contest_office_we_vote_id']
+                new_ballot_item.contest_measure_id = defaults['contest_measure_id']
+                new_ballot_item.contest_measure_we_vote_id = defaults['contest_measure_we_vote_id']
+                new_ballot_item.measure_subtitle = defaults['measure_subtitle']
+                new_ballot_item.polling_location_we_vote_id = defaults['polling_location_we_vote_id']
+                new_ballot_item.save()
+            else:
+                success = False
+                status = "BALLOT_ITEM_CREATE_FAILED"
+        except Exception as e:
+            success = False
+            new_ballot_item_created = False
+            status = "BALLOT_ITEM_RETRIEVE_ERROR"
+            handle_exception(e, logger=logger, exception_message=status)
+
+        results = {
+                'success':                      success,
+                'status':                       status,
+                'new_ballot_item_created':      new_ballot_item_created,
+                'ballot_item':                  new_ballot_item,
+            }
+        return results
+
+    def update_ballot_item_row_entry(self, ballot_item_display_name, local_ballot_order, state_code,
+                                     google_civic_election_id, defaults):
+        """
+        Update BallotItem table entry with matching we_vote_id
+        :param ballot_item_display_name:
+        :param local_ballot_order:
+        :param state_code:
+        :param google_civic_election_id:
+        :param defaults:
+        :return:
+        """
+
+        success = False
+        status = ""
+        ballot_item_updated = False
+        existing_ballot_item_entry = ''
+        ballot_item_found = False
+
+        try:
+            if positive_value_exists(defaults['contest_office_we_vote_id']):
+                existing_ballot_item_entry = BallotItem.objects.get(
```

Comment:
Hi @jainanisha90, we need to retrieve the existing BallotItem with a minimum of three values:
1) contest_office_we_vote_id or measure_we_vote_id
2) polling_location_we_vote_id
3) google_civic_election_id
Without all three we won't be able to find the unique ballot_item entry we are trying to update.